htdocs/resource/class/dolresource.class.php

```diff
@@ -171,7 +131,8 @@ public function __construct(DoliDb $db)
 	{
 		$this->db = $db;
 		$this->status = 0;
-
+		$this->element = 'dolresource';
```

Comment:
This properties are pseud constant. There is no reason to have them modified so they can be set statically and not intothe constructor.